rinja/src/dynamic/child.rs

```diff
@@ -0,0 +1,207 @@
+use std::io::ErrorKind;
+use std::process::exit;
+use std::string::String;
+use std::sync::Arc;
+use std::time::Duration;
+use std::vec::Vec;
+use std::{eprintln, format};
+
+use linkme::distributed_slice;
+use tokio::io::{AsyncBufReadExt, AsyncWriteExt, BufReader, Stdin, Stdout, stdin, stdout};
+use tokio::spawn;
+use tokio::sync::{Mutex, oneshot};
+
+use super::{DYNAMIC_ENVIRON_KEY, MainRequest, MainResponse, Outcome};
+
+const PROCESSORS: usize = 4;
```

Comment:
Any reason why four?

Reply:
[![](https://imgs.xkcd.com/comics/random_number.png)](https://xkcd.com/221/)  
No. :) Maybe the processor count would be better?

Reply:
Why do we need multiple processors?

Reply:
Without it the templates are rendered one after another.

On my not-so-recent AMD Ryzen 7 2700X system, with four workers and in release mode, the rendering takes about the same time in dynamic mode as non-dynamic: 68µs vs 65µs in he axum example.

Reply:
I see, another code comment. :D